_pages/en_US/installing-boot9strap-(frogtool).txt

```diff
@@ -24,7 +24,7 @@ To use the [magnet](https://wikipedia.org/wiki/Magnet_URI_scheme) links on this
 
 1. Insert your SD card into your computer
 1. Copy your `movable.sed` file to the root of your SD card
-1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+1. Copy `boot.firm` from the Luma3DS `.zip` to the root of your SD card
```

Comment:
the reason boot.3dsx is listed there is to ensure that someone has the latest version. since the change in PicHaxx guide suggested above tells them to skip to this guide if they have homebrew, they may be running an outdated version and would never be told to update it.